docs/usage/webworker.md

```diff
@@ -136,7 +136,7 @@ function requestResponse(worker, msg) {
     const { id, ...rest } = data;
     resolve(rest);
   });
-  worker.postMessage({ id, ...msg });
+  worker.postMessage({ idWorker, ...msg });
```

Comment:
Shouldn't this be

```suggestion
  worker.postMessage({ id: idWorker, ...msg });
```

?

Reply:
You are correct, and making that change actually highlighted there was a second bug with this example that needed to be fixed. It was not hit before, because the mismatch in id was causing it to return early. Both are now addressed by the changes.

To highlight the other issue with the example code, this was not valid from the original sample because there is no `data` variable:
```
const { id, ...rest } = data;
```
And instead needed to be this:
```
const { id, ...rest } = event.data;
```